### ISSUE TITLE
Can't resolve '../build/Release/lz4'

### ISSUE THREAD

**Issue by JosephP91** [pierrec/node-lz4#106]
Hello,

when I try to require the lz4 module in the browser, I get a compilation error:

`./node_modules/lz4/lib/utils.js
Module not found: Can't resolve '../build/Release/lz4' in '/Users/giuseppepersico/Github/perspective-test/node_modules/lz4/lib'`

Any suggestions? I am using the latest version of the dep, which is 0.6.5

**Comment by ankon** [comment]
I don't think that **node**-lz4 would be usable in a browser -- it needs the native extension of the lz4 library itself.

**Comment by nieyuyao** [comment]
You can use `lz4.js` under `build` directory of this repository in browser.

**Comment by ankon** [comment]
@nieyuyao have you done that successfully in a browser?

**Comment by nieyuyao** [comment]
> @nieyuyao have you done that successfully in a browser?

yes, i use `build/lz4.js` in browser.

**Comment by nieyuyao** [comment]
@ankon yes, i use `build/lz4.js` in browser.
